content/site.json

```diff
@@ -16,6 +16,6 @@
   ],
   "announcement": "[Lock your OCEAN](https://df.oceandao.org/) to get veOCEAN, earn rewards and curate data.",
   "warning": {
-    "ctd": "Compute-to-Data is still in a testing phase, please use it only on test networks."
+    "ctd": "Upload an algorithm using this feature. Please note that Compute-to-Data is still in alpha phase."
```

Comment:
```suggestion
    "ctd": "Please note that Compute-to-Data is still in alpha phase."
```

Reply:
@jamiehewitt15 I like your proposed change, and I also like the idea of removing the message completely. I am leaning toward the 2nd option because C2D has been in prod since 2019. Your thoughts?